modules/setting-up-vmc-for-vsphere.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vmc/installing-restricted-network-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+[id="setting-up-vmc-for-vsphere_{context}"]
+= Setting up VMC for vSphere
+
+You can install {product-title} on VMware Cloud (VMC) on AWS hosted vSphere clusters to enable applications to be deployed and managed both on-premises and off-premises, across the hybrid cloud.
+
+image::vmc-illustration.png["VMC on AWS Architecture"]
+
+You must configure several options in your VMware Cloud VMC environment prior to installing {product-title} on VMware vSphere. Ensure your VMC environment has the following prerequisites:
+
+* Create a non-exclusive, DHCP-enabled, NSX-T network segment and subnet. Other virtual machines (VM) can be hosted on the subnet, but at least eight IP addresses must be available for the {product-title} deployment.
+
+* Allocate two IP addresses, outside the DHCP range, and configure them with reverse DNS records.
+** A DNS record for `api.<cluster name>.<domain name>` pointing to the allocated IP address.
+** A DNS record for `*.apps.<cluster name>.<domain name>` pointing to the allocated IP address.
```

Comment:
Tow comments here:
1. It is better to change <domain name> to <base_domain>, which keep consistency with below DNS records chapter.
2. For IPI, we need two static IP address for api and apps DNS records. But I see this part is also in UPI installation part. For upi, both of them point to the the load balancer server if user has.

Reply:
>2. For IPI, we need two static IP address for api and apps DNS records. But I see this is also in UPI installation part. For upi, both of them point to the the load balancer server.

@jinyunma good point; I see the required DNS records for UPI are [here](https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-dns-user-infra_installing-vmc-user-infra). Should those six DNS record requirements for vSphere be documented in the VMS prereqs? Or should lines 21-23 here on the two static IP addresses and DNS records just be skipped for UPI?

Reply:
The requirement of DNS records for both upi and ipi are the same whether user deploy ocp in VMC or other vsphere platform, they should follow[ User-provisioned DNS requirements ](https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-dns-user-infra_installing-vmc-user-infra) and [DNS records for IPI](https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc.html#installation-vsphere-installer-infra-requirements_installing-vmc), I checked your update, it's LGTM to skip them in VMC prereqs.